packages/core/src/services/exports/findByUuid.ts

```diff
@@ -0,0 +1,24 @@
+import { database } from '@latitude-data/core/client'
+import { latitudeExports } from '../../schema/models/exports'
+import { and, eq } from 'drizzle-orm'
+import { Export } from '../../browser'
+
+export async function findByUuid(
+  exportUuid: string,
+): Promise<Export | undefined> {
+  return await database.query.latitudeExports.findFirst({
+    where: eq(latitudeExports.uuid, exportUuid),
+  })
+}
+
+export async function findByUuidAndWorkspace(
+  uuid: string,
+  workspaceId: number,
+): Promise<Export | undefined> {
+  return await database.query.latitudeExports.findFirst({
+    where: and(
+      eq(latitudeExports.uuid, uuid),
+      eq(latitudeExports.workspaceId, workspaceId),
+    ),
+  })
+}
```

Comment:
Why not a collection? I wouldn't start another way of doing things while we have already one working

Reply:
Also a service is weird only for getting data

Reply:
ok

Reply:
Another tradeoff I see in putting all in `services/**/*.ts` is that making tests for each data access file will fill the folder with files. I like services to be only things that do one thing, writing data. 
At least put all data access in `data-access/**/*.ts`. Although I think should investigate again if Drizzle now supports a kind of tenancy that could work for us